src/main/java/com/rewedigital/composer/composing/IncludedService.java

```diff
@@ -86,24 +88,40 @@ private IncludedService(final Builder builder) {
         this.fallback = builder.fallback;
     }
 
-
     public CompletableFuture<IncludedService.WithResponse> fetch(final ContentFetcher fetcher,
         final CompositionStep parentStep) {
         final CompositionStep step = parentStep.childWith(path());
-        return fetcher.fetch(path(), fallback(), step)
+        return fetcher.fetch(FetchContext.of(path(), fallback(), ttl()), step)
             .thenApply(r -> new WithResponse(step, startOffset, endOffset, r));
     }
 
-    private String fallback() {
+    public boolean isInRage(final ContentRange contentRange) {
+        return contentRange.isInRange(startOffset);
+    }
+
+    String fallback() {
         return fallback;
     }
 
-    private String path() {

```

Comment:
I suggest making these methods `public` instead of package private, if they must be accessible now

Reply:
Done

Reply:
Annotated with VisibleForTesting